Analysis/config/2017/sources.cfg

```diff
@@ -1,40 +1,42 @@
 [ANA_DESC common]
 int_lumi: 41529.343
 period: Run2017
-final_variables: m_ttbb_kinfit MT2 mva_score
 energy_scales: all
 #categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF
 #categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF 2j1b+B 2j1b+B_noVBF 2j0b 2j1b 2j2b+ 2j0Lb 2j1Lb 2j2Lb 2j1bR 2j2b+R
 categories: 2j
-sub_categories: NoCuts
-#sub_categories: mh
+#sub_categories: NoCuts
+sub_categories: mh
 regions: OS_Isolated OS_AntiIsolated SS_Isolated SS_AntiIsolated SS_LooseIsolated
 limit_setup: MVA_res mva_score:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF  m_ttbb_kinfit:4j1b+_VBF
 limit_setup: MVA_nonres mva_score:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF MT2:4j1b+_VBF
+#limit_setup: Sync dR_l1l2:2j
+limit_setup: Sync pt_H_tt:2j
 hist_cfg: hh-bbtautau/Analysis/config/2017/histograms.cfg
 plot_cfg: hh-bbtautau/Analysis/config/2017/plot_setup.cfg
 plot_page_opt: internal_plot
 massWindowParams: mh 116.0 35.0 111.0 45.0
 massWindowParams: mhVis 87.9563 41.8451 109.639 43.0346
 jet_ordering: DeepCSV
 #unc_cfg: hh-bbtautau/Analysis/config/2017/prefit_unc.cfg
-syncDataIds: 2j/NoCuts/OS_Isolated/Central/DY_nlo
+#syncDataIds: 2j/NoCuts/OS_Isolated/Central/DY_nlo
+#syncDataIds: 2j/NoCuts/OS_Isolated/Central/TTTo2L2Nu
 #syncDataIds: 2j/NoCuts/OS_Isolated/Central/ 2jets1btagR/mh/OS_Isolated/Central/TT 2jets2btagR/mh/OS_Isolated/Central/TT 2jets/mh/OS_Isolated/Central/Data_Tau 2jets1btagR/mh/OS_Isolated/Central/Data_Tau 2jets2btagR/mh/OS_Isolated/Central/Data_Tau
 trigger: eTau HLT_Ele32_WPTight_Gsf_v HLT_Ele35_WPTight_Gsf_v HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTau30_eta2p1_CrossL1_v
 trigger: muTau HLT_IsoMu24_v HLT_IsoMu27_v HLT_IsoMu20_eta2p1_LooseChargedIsoPFTau27_eta2p1_CrossL1_v
 trigger: muMu HLT_IsoMu24_v HLT_IsoMu27_v
 trigger: tauTau HLT_DoubleTightChargedIsoPFTau35_Trk1_TightID_eta2p1_Reg_v HLT_DoubleMediumChargedIsoPFTau40_Trk1_TightID_eta2p1_Reg_v HLT_DoubleTightChargedIsoPFTau40_Trk1_eta2p1_Reg_v
 
 [ANA_DESC full : common]
-signals: GluGluSignal_Radion GluGluSignal_Graviton GluGluSignal_NonRes
+#signals: GluGluSignal_Radion GluGluSignal_Graviton GluGluSignal_NonRes
 signals: GluGluSignal_NonRes VBFSignal_NonRes
-backgrounds: TTTo2L2Nu TTToSemiLeptonic TTToHadronic DY_nlo Wjets VV VH EWK ST QCD
+backgrounds: TTTo2L2Nu TTToSemiLeptonic TTToHadronic DY_nlo Wjets VV VVV VH EWK ST QCD ttV ttVV SM_Higgs ttH
 #backgrounds: DY_nlo
 data: Data_SingleElectron Data_SingleMuon Data_Tau
 #data: Data_Tau
-cmb_samples: other_bkg
-#draw_sequence: DY_nlo
-draw_sequence: data TTTo2L2Nu TTToSemiLeptonic TTToHadronic DY_nlo QCD Wjets other_bkg VH signals
+cmb_samples: other_bkg TT SM
+#draw_sequence: data DY_nlo TT Wjets QCD EWK ST SM VVV ttV ttVV VV
+draw_sequence: data TT DY_nlo  QCD Wjets other_bkg VH signals
```

Comment:
Please, control that all backgrounds are considered when drawing. I.e. it should be SM instead of VH, other_bkg should include all newlly introduce sample descriptors, etc. Also check that colors in the plots are fine.